packages/tools/client-debugger/client-debugger-view/src/components/TelemetryView.tsx

```diff
@@ -99,6 +124,28 @@ export function TelemetryView(): React.ReactElement {
 		};
 	}, [messageRelay, setTelemetryEvents]);
 
+	/**
+	 * Filters all telemetry events based on category and event name
+	 * @returns filtered list of events
+	 */
+	function getFilteredEvents(): ITimestampedTelemetryEvent[] | undefined {
+		let filteredEvents = telemetryEvents;
```

Comment:
Is this line of code necessary?

Reply:
Yes I initialize filteredEvents to all telemetryEvents. This would represent the initial state when the user has not set any filters.